.changeset/curly-peas-peel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': patch
+---
+
+Table's Ctrl+Shift click implementation more consistent with Windows Explorer's implementation.
```

Comment:
nit: missing verb here

e.g. "**Made** Table's Ctrl+Shift click…"

Reply:
I thought to leave it out to be concise. But yes, adding that one word makes it more clearer. Added (https://github.com/iTwin/iTwinUI/pull/1660/commits/8aa84ef16b1a01bebde9047332ee0e8b495d8e98).